docs/sdk/client-side/javascript/index.md

```diff
@@ -55,13 +55,17 @@ Configure the SDK config and user configuration.
 <TabItem value="js" label="Javascript">
 
 ```js showLineNumbers
-const config = {
+const config = defineBKTConfig({
   apiKey: 'YOUR_API_KEY',
-  apiURL: 'YOUR_API_URL',
-  featureTag: 'YOUR_FEATURE_TAG'
-};
-
-const user = {id: 'USER_ID'};
+  apiEndpoint: 'YOUR_API_URL',
+  featureTag: 'YOUR_FEATURE_TAG',
+  appVersion: 'YOUR_APP_VERSION',
+  fetch: window.fetch // your fetch implementation
```

Comment:
Is this required for the initialization?

Reply:
Let's make this optional, then.
Btw, does the old SDK have such implementation internally?
I'm not sure what it does exactly.

Reply:
https://github.com/ca-dp/bucketeer-web-sdk/blob/master/src/index.ts#L37

`fetch` is required field in the old SDK.

Reply:
created a PR: https://github.com/bucketeer-io/javascript-client-sdk/pull/54
also fixed the document in [`db25a11` (#11)](https://github.com/bucketeer-io/bucketeer-docs/pull/11/commits/db25a11e2f6387ac98033121bc9ae25afde31edf)